pkg/plugins/policies/core/defaults/consts.go

```diff
@@ -10,6 +10,11 @@ const (
 	DefaultIdleTimeout           = time.Hour
 	DefaultStreamIdleTimeout     = 30 * time.Minute
 	DefaultRequestTimeout        = 15 * time.Second
+	DefaultRequestHeadersTimeout = 0
 	DefaultMaxStreamDuration     = 0
 	DefaultMaxConnectionDuration = 0
+	// Gateway
+	DefaultGatewayIdleTimeout           = 5 * time.Minute
+	DefaultGatewayStreamIdleTimeout     = 5 * time.Second
+	DefaultGatewayRequestHeadersTimeout = 500 * time.Millisecond
```

Comment:
I think if we are putting these defaults here, we should get rid of these and just reuse ones you just added https://github.com/kumahq/kuma/blob/8cc520ea4c14e20b386702fcf41056c66d44f33b/pkg/plugins/runtime/gateway/filter_chain_generator.go#L56-L58